backend/app/views/spree/admin/promotions/duplicate.html.erb

```diff
@@ -0,0 +1,96 @@
+<% content_for :page_title do %>
+  <%= Spree.t(:duplicating_promotion) %>
+<% end %>
+
+<%= render 'spree/admin/shared/promotion_sub_menu' %>
+
+<% content_for :page_actions do %>
+  <li>
+    <%= button_link_to Spree.t(:back_to_promotions_list), admin_promotions_path, :icon => 'arrow-left' %>
+  </li>
+<% end %>
+
+<%= form_for @promotion, :url => admin_promotions_path, :method => :post do |f| %>
+  <fieldset class="no-border-top">
+    <%= render :partial => 'form', :locals => { :f => f } %>
+    <%= render :partial => 'spree/admin/shared/new_resource_links' %>
+  </fieldset>
+
+  <div id="duplicate-promotions-overlay">
+    <div id="promotion-filters" class="row">
+      <div id="rules_container" class="alpha eight columns">
+
+        <fieldset id="rule_fields" class ="alpha eight columns no-border-bottom no-border-top">
+          <fieldset>
+            <legend align="center"><%= Spree.t(:rules) %></legend>
+            <% if @promotion.rules.any? %>
+              <div class="field">
+                <%= label_tag :promotion_rule_type, Spree.t(:add_rule_of_type) %>
+                <%= select_tag('promotion_rule[type]', options_for_promotion_rule_types(@promotion), :class => 'select2 fullwidth') %>
+              </div>
+          </fieldset>
+          <fieldset class="no-border-top">
+            <div id="promotion-pilicy-select" class="align-center row">
+              <% Spree::Promotion::MATCH_POLICIES.each do |policy| %>
+                <div class="alpha four columns">
+                  <label><%= f.radio_button :match_policy, policy %> <%= Spree.t "promotion_form.match_policies.#{policy}" %></label>
+                </div>
+              <% end %>
+            </div>
+
+            <div id="rules" class="filter_list row">
+              <% @promotion.rules.each do |promotion_rule| %>
```

Comment:
They exist as partials currently, but there is a lot of picking and choosing what I wanted to the point of not seeing a way to separate them out in a way that really made sense. I could make a bunch of really small partials and redo all of that?